frontend/src/css/main.scss

```diff
@@ -155,6 +155,28 @@ input.ng-dirty.ng-invalid, select.ng-dirty.ng-invalid {
     }
 }
 
```

Comment:
1. we're using bootstrap-css so if there are any stuff there to use that should be used before creating our own css classes.
2. If there is nothing there new classes can be made, but it is advisable to create a parent container for them in the html and then use scss nesting to add them.
3. Maybe we can take it as is and beautify later on.